It's because:
```
./addons/sale/migrations/10.0.1.0/pre-migration.py:89:66: W291 trailing whitespace
```
You have an space at the end of the line. Configure your IDE to remove them automatically.

About the best query, `IN` or `= ANY`, the final conclusion is that it performs the same (IN is transformed to = ANY), so the only advantage of ANY is to not convert everything to tuples.

--- a/addons/sale/migrations/10.0.1.0/pre-migration.py
+++ b/addons/sale/migrations/10.0.1.0/pre-migration.py
@@ -86,8 +86,8 @@ def warning_update_module_names_partial(cr):
         return
 
     # get moved model fields
-    moved_fields = tuple(['sale_line_warn', 'sale_line_warn_msg', 'sale_warn',
-                    'sale_warn_msg'])
+    moved_fields = tuple(['sale_line_warn', 'sale_line_warn_msg',
+                          'sale_warn', 'sale_warn_msg'])
     cr.execute("""
         SELECT id
         FROM ir_model_fields

